db/migrate/20180624155644_add_show_dinkurs_events_column_to_company.rb

```diff
@@ -0,0 +1,8 @@
+class AddShowDinkursEventsColumnToCompany < ActiveRecord::Migration[5.1]
+  def change
+    add_column :companies, :show_dinkurs_events, :boolean
+    Company.find_each do |company|
+      company.show_dinkurs_events = !company.dinkurs_company_id.blank?
```

Comment:
Sorry - I should have caught this before .... you're not saving the updated company record here.

So, you could just add the statement `company.save` to the block.

It's interesting as to whether the `find_each` method is faster than a more direct approach, such as:
```
Company.where.not(dinkurs_company_id: [nil, '']).each { |c| c.update_attribute(:show_dinkurs_events, true) }
```
`find_each` will pass _every_ company to the block.  The alternative above only passes the companies with a non-blank dinkurs key, but involves more SQL.

Our DB is small enough that it really doesn't matter ... just wondering about this (your logic is fine - no change needed except for the `save`).

Reply:
On second thought, I'd recommend changing to the SQL-select approach - that is a better approach overall and would make a significant performance difference on a large DB.